riscv/csrs.cc

```diff
@@ -20,6 +20,8 @@
 #undef STATE
 #define STATE (*state)
 
+#define CBIE_RESERVED 2 // Reserved value of xenvcfg.CBIE
```

Comment:
When I [asked](https://github.com/riscv-software-src/riscv-isa-sim/pull/1416#discussion_r1269366332) to make this a macro, I explained my reasoning: so it would be next to the other CBIE definitions.

But now I see that there are no other CBIE definitions. (Is that right? Are the interpretations of 0/1/3 defined anywhere? I can't find any.)

In that case, this is better the previous way (as a local `const`).

Reply:
Sorry to go back and forth like this. I was assuming last week that somewhere else we had something like:
```
#define CBIE_FOO 0
#define CBIE_BAR 1
#define CBIE_FOOBAR 3
```

Reply:
To my knowledge, there is no public header file that includes all definitions of CSR values. Although the ```debug_defines.h``` includes the value definitions of debug CSRs, it does not include other CSRs. The other possible public header file for global definitions of CSR values is ```encoding.h```, which also includes some value definitions of debug CSRs. However, I believe the ```encoding.h``` should only contain the definition of CSR fields instead of values. 
Nevertheless, I agree there should be a public header file that includes all definitions of CSR values. By the naming rule in ```debug_defines.h``` and ```encoding.h```, we should have something like:
```
#define MENVCFG_CBIE_EXCEPTION 0
#define MENVCFG_CBIE_FLUSH 1
#define MENVCFG_CBIE_RESERVED 2
#define MENVCFG_CBIE_INVALIDATE 3
```
![image](https://github.com/riscv-software-src/riscv-isa-sim/assets/39526191/7b647cad-e781-4760-8108-004d90067bc1)